README.md

```diff
@@ -70,8 +70,7 @@ See [`defaults/main.yml`](defaults/main.yml). Required variables:
 * [`geerlingguy.nginx`](https://galaxy.ansible.com/geerlingguy/nginx/)
 * [`geerlingguy.repo-epel`](https://galaxy.ansible.com/geerlingguy/repo-epel/)
 * [`jdauphant.ssl-certs`](https://galaxy.ansible.com/jdauphant/ssl-certs/)
-* [`williamyeh.oracle-java`](https://galaxy.ansible.com/williamyeh/oracle-java/)
-    * Using [a fork](https://github.com/gjedeer/ansible-oracle-java), due to [this issue](https://github.com/William-Yeh/ansible-oracle-java/issues/58)
+* [`srsp.oracle-java`](https://galaxy.ansible.com/srsp/oracle-java/)
```

Comment:
Ha, wow, the errors earlier were because `git` wasn't installed in the container.

![facepalm](http://gif-finder.com/wp-content/uploads/2015/02/Steve-Carell-Facepalm.gif)